Formula/aws-amplify.rb

```diff
@@ -0,0 +1,23 @@
+require "language/node"
+
+class AwsAmplify < Formula
+  desc "Build full-stack web and mobile apps in hours. Easy to start, easy to scale"
+  homepage "https://aws.amazon.com/amplify"
+  url "https://registry.npmjs.org/@aws-amplify/cli/-/cli-12.0.0.tgz"
+  sha256 "ad1ce490803b65342736b98985cdd0682ddfce6927ed839073bf80576955ce79"
+  license "Apache-2.0"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Replace universal binaries with native slices.
+    deuniversalize_machos
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/amplify --version")
```

Comment:
but the test cases did not show up in here? i think we can keep that negative credentials test

Reply:
<img width="1411" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/7275620/146ab42c-aaeb-407a-b76c-9451543f9c82">

Same problem

Reply:
<img width="1146" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/7275620/d7dab796-5ea8-4c5f-8e20-609f057fe281">

@chenrui333 Do you know why is the ci job not running?

Reply:
However I modify the test cases, and it always can't pass in macOS 12-arm64 and macOS 13-arm64, I don't know why, but it can run in my local environment.

Reply:
interesting.